README.md

```diff
@@ -34,7 +34,8 @@ Prequisites:
 - `gem install foreman`
 - `bundle install`
 - `yarn install`
-- `cp .env.local.sample .env` -- fill in env vars where applicable
+- `cp .env.sample .env` -- dotenv loads variables from a **.env** in your project's root directory. Fill in env vars where applicable
+- `cp .env.local .env` -- for people with a .env.local
```

Comment:
This seems confusing. Do we need both of these? Can we consolidate them into one file?

Reply:
Yeah, my bad. Let's just keep the first line that tells them to copy the contents from .env.sample to a .env